### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by a-jakubowska**
updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,21 @@
 # mtw2-translator
 Translator for Medieval Total War II mods
 
-## Usage
+#Usage
```
Comment (pawel-jakubowski) [inline README.md]: needs space between # and Usage

File: README.md [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
+
+Steps to Follow:
+
+1. Optional - Install Pthon version
+2. Download and run mtw2-translator exe file
+4. In opened console type language and folder name (see example)
+5. In case you have any referance translation, you can also use them
+6. Warning! Program is using web version of Google Translator,it can take long time to process.
```
Comment (pawel-jakubowski) [inline README.md]: Please use :exclamation: instead (nicer output)

**Review by pawel-jakubowski — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (pawel-jakubowski) [reply README.md]: `:exclamation:`

Comment (a-jakubowska) [reply README.md]: done, thanks :)

File: README.md (hunk 1 above)
Comment (a-jakubowska) [reply README.md]: done

**Review by pawel-jakubowski — APPROVED** [review]

**Pull request merged**